### DISCUSSION TITLE
Podcast on Terminal.Gui

### DISCUSSION THREAD

**Discussion by tig** [tui-cs/Terminal.Gui#1350] [General]
Check out this podcast where I discuss Terminal.Gui (aka gui.cs).

http://azuredevopspodcast.clear-measure.com/charlie-kindel-on-terminalgui-episode-146

**Comment by BDisp** [comment]
I love it. Thanks.

**Reply by tznind** [reply]
Yeah great interview, good topic coverage and engaging :)
